ca/server.go

```diff
@@ -530,13 +534,8 @@ func (s *Server) evaluateAndSignNodeCert(ctx context.Context, node *api.Node) {
 
 // signNodeCert does the bulk of the work for signing a certificate
 func (s *Server) signNodeCert(ctx context.Context, node *api.Node) {
-	if !s.securityConfig.RootCA().CanSign() {
-		log.G(ctx).WithFields(logrus.Fields{
-			"node.id": node.ID,
-			"method":  "(*Server).signNodeCert",
-		}).Errorf("no valid signer found")
-		return
-	}
+	rootCA := s.securityConfig.RootCA()
+	externalCA := s.securityConfig.ExternalCA()
```

Comment:
OK, When the `MutableTLSCreds` in the `SecurityConfig` change, is there a function somewhere that I can hook into? At the very least, I need to recreate it whenever the TLS config changes (due to node cert renewal) and when the cluster config changes (to get a possible updated list of external CA URLs).